packages/discovery/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@l2beat/discovery",
+  "version": "0.1.0",
+  "private": true,
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "bin": {
+    "discovery": "./cli.js"
+  },
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
+    "clean": "rm -rf build",
+    "format:fix": "prettier --write .",
+    "format": "prettier --check .",
+    "lint:fix": "yarn lint --fix",
+    "lint": "eslint --ext .ts --max-warnings 0 src",
+    "test": "mocha",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@l2beat/backend-tools": "*",
+    "@l2beat/discovery-types": "^0.1.0",
```

Comment:
Please run `yarn changeset` and bump `@l2beat/backend-tools` and `l2beat/discovery-types` minor versions. Specify the versions here.

Reply:
Bumping discovery types is not required - I reverted exporting `EthereumAddress` and `Hash256` from barrel files since we already have duplicates in place until we resolve those. 

https://github.com/l2beat/tools/pull/25/commits/65066fd032013a21948631063a3578e21a08a7fd